src/org/opendatakit/common/cli/Operation.java

```diff
@@ -39,18 +39,6 @@ public static Operation of(Param param, Consumer<Args> argsConsumer) {
     return new Operation(param, argsConsumer, emptySet(), emptySet(), false);
   }
 
-  /**
-   * Creates a new {@link Operation} with some required params
-   *
-   * @param param          main {@link Param} that will trigger the execution of this {@link Operation}, normally a {@link Param#flag(String, String, String)}
-   * @param argsConsumer   {@link Consumer}&lt;{@link Args}&gt; with the logic of this {@link Operation}
-   * @param requiredParams a {@link Set}&lt;{@link Param}&gt; with the required params for this operation
-   * @return a new {@link Operation} instance
-   */
-  public static Operation of(Param param, Consumer<Args> argsConsumer, List<Param> requiredParams) {
```

Comment:
new/revisit

This class is an internal library. I think having non-used methods is not so important.